src/templates/RoundedActionBox/RoundedActionBox.tsx

```diff
@@ -0,0 +1,83 @@
+import { type ReactNode } from "react";
+import { Box, BoxProps } from "~/components";
+import { roundedActionBoxRecipe } from "./RoundedActionBox.css";
+
+export const RoundedBox = ({
+  children,
+  disabled = false,
+  error = false,
+  ...boxProps
+}: BoxProps & {
+  disabled?: boolean;
+  error?: boolean;
+}) => {
+  return (
+    <Box
+      {...boxProps}
+      display="flex"
+      flexDirection="column"
+      rowGap={0}
+      borderWidth={1}
+      borderStyle="solid"
+      borderRadius={5}
+      className={roundedActionBoxRecipe({ disabled, error })}
+    >
+      {children}
+    </Box>
+  );
+};
+
+export const RoundedBoxWithFooter = ({
```

Comment:
I dont think there is a need to make it a separate component.

How about one of these:
1. `<RoundedBox footer={...} />` - footer will be shown if provided, otherwise ignored
2. `<RoundedBox><div>Content</div> <RoundedBox.Footer> <Button>Save</Button> </RoundedBox.Footer> </RoundedBox>`